spectec/spec/wasm-1.0/9-module.watsup

```diff
@@ -129,31 +129,40 @@ def $concat_instr((instr*)*) : instr*
 def $concat_instr(epsilon) = epsilon
 def $concat_instr((instr*) (instr'*)*) = instr* $concat_instr((instr'*)*)
 
-def $initelem(store, tableaddr, u32*, (funcaddr*)*) : store
-def $initelem(s, ta, epsilon, epsilon) = s
-def $initelem(s, ta, i i'*, (a*) (a'*)*) = s[.TABLE[ta].ELEM[i:|a*|] = a*]
+def $initelem(store, moduleinst, u32*, (funcaddr*)*) : store
+def $initelem(s, mm, epsilon, epsilon) = s
+def $initelem(s, mm, i i'*, (a*) (a'*)*) = s_2
+  -- if s_1 = s[.TABLE[mm.TABLE[0]].ELEM[i:|a*|] = a*]
+  -- if s_2 = $initelem(s_1, mm, i'*, (a'*)*)
 
-def $initdata(store, memaddr, u32*, (byte*)*) : store
-def $initdata(s, ma, epsilon, epsilon) = s
-def $initdata(s, ma, i i'*, (b*) (b'*)*) = s[.MEM[ma].DATA[i:|b*|] = b*]
+def $initdata(store, moduleinst, u32*, (byte*)*) : store
+def $initdata(s, mm, epsilon, epsilon) = s
+def $initdata(s, mm, i i'*, (b*) (b'*)*) = s_2
+  -- if s_1 = s[.MEM[mm.MEM[0]].DATA[i:|b*|] = b*]
+  -- if s_2 = $initdata(s_1, mm, i'*, (b'*)*)
 
 def $instantiate(store, module, externval*) : config
-def $instantiate(s, module, externval*) = s'; f; (CALL x')?
+def $instantiate(s, module, externval*) = s'''; f; (CALL x')?
```

Comment:
The store should be propagated through initializing elems and datas, thus `s'''` here.
(Perhaps `s_3` will be better?)

Reply:
I'm fine with changing this to s_N.